scripts/ci/jobs/diff-dumps.sh

```diff
@@ -100,7 +100,7 @@ diff_dumps() {
 
     # These are not needed until later, but no reason to continue if these do not exist.
     require_environment "GOOGLE_SA_STACKROX_HUB_VULN_DUMP_UPLOADER"
-    require_environment "GCP_SERVICE_ACCOUNT_CREDS"
+    require_environment "SCANNER_GCP_SERVICE_ACCOUNT_CREDS"
```

Comment:
assuming you added this to the repo?

Reply:
That's correct - added at the organization level, then configured for this repo. The environment variables with the corresponding secrets are set in the workflow.yaml.

Reply:
We could also not change the name here and instead bind the secret `secrets.SCANNER_GCP_SERVICE_ACCOUNT_CREDS` to the environment variable `GCP_SERVICE_ACCOUNT_CREDS`